packages/x-components/src/x-modules/tagging/wiring.ts

```diff
@@ -255,14 +255,14 @@ export const trackNoResultsQueryWithSemanticsWire = filter(
 )
 
 /**.
- * Debounced version of {@link trackNoResultsQueryWithSemanticsWire}
+ * Debounced version of {@link trackNoResultsQueryWithFallbackWire}
  *
  * @public
  */
-export const trackNoResultsQueryWithSemanticsWireDebounced = moduleDebounce(
-  trackNoResultsQueryWithSemanticsWire,
+export const trackNoResultsQueryWithFallbackWireDebounced = moduleDebounce(
```

Comment:
idem here, should we track BreakingChange and inform that we are adding a new cancel event?